README.md

```diff
@@ -32,6 +32,10 @@ Question keys
     * `"on"` service data key name to use for comparison (e.g. "lot" for lots)
     * `being` a list of acceptable values for the key. If service data key value matches one of the values in the `being` the
        question is kept, otherwise the question is removed from the section for the given service
+* `limits` a dictionary of validation/type requirements to limit the possible question values. Available fields:
+    * `min_value` integer, sets the minimum value for percentage questions
+    * `max_value` integer, sets the maximum value for percentage questions
+    * `integer_only` boolean, when `true` makes the percentage question accept only integer values instead of numbers
```

Comment:
The idea behind `limits` is to group all validation-related fields together. These are used primarily by the schema generation and can apply to questions of all types. I'm not attached to this nested structure since it doesn't add that much in the end, but it feels a bit cleaner than mixing all the fields in one question namespace.

I'm not sure what you mean by "in which case min_value seems a bit vague"?

Limit fields themselves could be type-specific or more general, so if `min_value` makes sense for other field types it should be fine to reuse it. The documentation would have to be updated if that happens.